opencensus/stats/exporters/stackdriver_exporter.py

```diff
@@ -0,0 +1,55 @@
+import os
+from google.cloud import monitoring
+from google.cloud.monitoring import Client
+from google.cloud.monitoring import Metric
+from google.cloud.monitoring import MetricDescriptor
+from google.cloud.monitoring import MetricKind, ValueType
+from google.cloud.monitoring import LabelValueType
+from google.cloud.monitoring import LabelDescriptor
+from google.cloud.monitoring import Resource
+from opencensus.stats import view
+
+class StackDriverExporter(object):
+    def __init__(self, client=None, project_id=None, resource=None):
+        if client is None:
+            client = Client(project=project_id)
+
+        self.client = client
+        self.project_id = client.project
+        self.resource = client.resource('global', {})
+
+    def emit(self, views, datapoint=None):
+        name = 'projects/{}'.format(self.project_id)
+        '''metric = {}'''
+        self.resource = self.set_resource(type_='global', labels= {'project_id': self.project_id})
+        metrics = self.translate_to_stackdriver(views)
+        for type, label in metrics:
```

Comment:
Should be `for metric_type, metric_label in metrics.items()`, `type` is a reserved keyword in python and please avoid using is as a variable name. And iterating on `metrics` (which is a dict) will only iterating on the keys, instead of the key value pairs.